src/gatsby-node.js

```diff
@@ -81,7 +81,6 @@ exports.sourceNodes = (
         dty_sub: '',
         web_address: '',
         intellcont_auth: {
-          faculty_name: '',
```

Comment:
I don't think we should remove this yet, because `master` of faculty-profiles mentions this key on line 146 of `gatsby_fp/src/components/Layout.js`